swaybar/status_line.c

```diff
@@ -186,6 +186,7 @@ struct status_line *status_line_init(char *cmd) {
 void status_line_free(struct status_line *status) {
 	status_line_close_fds(status);
 	kill(status->pid, SIGTERM);
+	kill(status->pid, SIGCONT);
```

Comment:
Can we send `status->cont_signal` instead of `SIGCONT`? It seems like the continuation signal is configurable.

Nit: I'd argue it makes a little bit more sense to send `SIGCONT` before `SIGTERM`.

Reply:
I'm going to unresolve this conversation since it looks like the following still needs to be addressed:

> Nit: I'd argue it makes a little bit more sense to send SIGCONT before SIGTERM

Reply:
Thanks for pointing it out!